target/linux/mediatek/dts/mt7981b-edup-rt2980.dts

```diff
@@ -250,6 +202,40 @@
 	};
 };
 
+&switch {
+	ports {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		port@0 {
+			reg = <0>;
+			label = "lan1";
+		};
+
+		port@1 {
+			reg = <1>;
+			label = "lan2";
+		};
+
+		port@2 {
+			reg = <2>;
+			label = "lan3";
+		};
+
```

Comment:
how does this work without the WAN port definition?
port 6 is internal port. port 3 should be the actual WAN.